pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -979,7 +981,32 @@ public void testLookupThrottlingForClientByClient() throws Exception {
         EventLoopGroup eventLoop = EventLoopUtil.newEventLoopGroup(20, false,
                 new DefaultThreadFactory("test-pool", Thread.currentThread().isDaemon()));
         long reqId = 0xdeadbeef;
-        try (ConnectionPool pool = new ConnectionPool(conf, eventLoop)) {
+
+        Semaphore clientCnxSemaphore = new Semaphore(1);
+        AtomicInteger count = new AtomicInteger(2);
```

Comment:
Why do we need this `count` ?

Reply:
After I thinking about this, there is no need to add `count` @poorbarcode , I will remove it.